packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_navigation_delegate.dart

```diff
@@ -128,6 +132,16 @@ abstract class PlatformNavigationDelegate extends PlatformInterface {
         'setOnWebResourceError is not implemented on the current platform.');
   }
 
+  /// Notify host application that the given WebView's render process has exited.
```

Comment:
Please use the same "Invoked when ..." structure as all the other comments here.